drivers/ws281x/include/ws281x_constants.h

```diff
@@ -27,6 +27,27 @@ extern "C" {
  * @name    Timing parameters for WS2812/SK6812 RGB LEDs
  * @{
  */
+
+/**
+ * @brief   Data transmission time in nanoseconds
+ *
+ * For the SK6812, WS2812 and WS2812b this is 1.25 µs. This is the total time
+ * required to transmit one bit.
+ */
+#define WS281X_T_DATA_NS                (1250U)
+
+/**
+ * @brief The high-times in nanoseconds.
+ *
+ * The WS2812b is incompatible with the timing of the WS2812 and SK6812.
+ * @{
+ */
+#define WS281X_T_DATA_ONE_NS            (650U)
+#define WS281X_T_DATA_ZERO_NS           (325U)
+#define WS2812B_T_DATA_ONE_NS           (800U)
+#define WS2812B_T_DATA_ZERO_NS          (450U)
+/**@}*/
```

Comment:
> The WS2812b is incompatible with the timing of the WS2812 and SK6812.

This is not correct. The tolerance of the times are +/- 150 ns, so that they overlap. Just take the average (as the ATmega driver does) and it will just work for both.